homeassistant/components/webostv/__init__.py

```diff
@@ -16,6 +16,7 @@
 )
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.dispatcher import async_dispatcher_send
+from .media_player import ATTR_SOUND_OUTPUT
```

Comment:
I think this will be a cyclic import. Put it in the init module or in a const.py module in the package.